addons/mail/tests/test_mail_channel.py

```diff
@@ -197,6 +201,27 @@ def test_channel_members(self):
         self.assertEqual(channel.message_partner_ids, self.env['res.partner'])
         self.assertEqual(channel.channel_partner_ids, self.env['res.partner'])
 
+    @users('employee')
+    @mute_logger('odoo.addons.mail.models.mail_mail', 'odoo.models.unlink')
+    def test_channel_chat_message_post_should_update_last_activity_time(self):
+        channel = self.env['mail.channel'].browse(self.test_channel.ids)
+        channel.write({'channel_type': 'chat'})
```

Comment:
Can't you find another demo channel that is already chat though? Or just create it if it's missing, it's strange to update it like that.

Reply:
It would be fine to just use the "get chat" method actually.

Reply:
a new chat channel is created in `setUpClass`